app.js

```diff
@@ -194,7 +188,7 @@ const parseCommandLineArguments = () => {
         return arg.split('=');
     }).forEach(arg => {
         // array value
-        if ( arg[1].includes('[') ) {
+        if (arg[1].includes('[')) {
             cmdArgs[arg[0]] = JSON.parse(arg[1]);
         } else {
             cmdArgs[arg[0]] = arg[1]
```

Comment:
@woskaniian all the changes in this file are not in a scope of this PR)
and also code style changes should be discussed and they are not something related to what is called `fix`
also the uncommenting can be done on you side locally at this moment